plugins/jobs/web_client/views/JobListWidget.js

```diff
@@ -28,6 +29,29 @@ var JobListWidget = View.extend({
         'change select.g-page-size': function (e) {
             this.collection.pageLimit = parseInt($(e.target).val());
             this.collection.fetch({}, true);
+        },
+        'change input.g-job-checkbox': function (e) {
+            var jobId = $(e.target).closest('tr').attr('g-job-id');
+            if ($(e.target).is(':checked')) {
```

Comment:
You usually want to use `e.currentTarget`, to ensure that you're selecting the element this handler was bound to, if the original event fired on a sub-element.